### ISSUE TITLE
Cant scroll down when magnify is on full screen image.

### ISSUE THREAD

**Issue by EvanCowan** [thdoan/magnify#55]
Im using magnify on large images on my site. It works well, but on mobile, the lens appears when i try and drag the page down, instead of the page scrolling down. Id rather just have a tap to enlarge on mobile without the lens, which Im hoping will sort the scrolling issue. Is that possible?

**Comment by thdoan** [comment]
Have you tried the mobile plugin? It is meant to be used on mobile (the main plugin is meant for desktops): 

https://thdoan.github.io/magnify/demo-mobile.html

**Comment by EvanCowan** [comment]
Yeah, I have the mobile min js loaded, but do I need to do something else?
Preview a page here: https://www1.sentinelhomes.co.nz/floor-plans-2a
I see the lens come up on android chrome.

**Comment by benlau87** [comment]
Same problem here. Mobile plugin loaded, but scrolling not possible when the image covers the whole screen :-(

**Comment by thdoan** [comment]
This is the mobile plugin's intended behavior. To scroll, close the overlay.

**Comment by benlau87** [comment]
But how do you close the overlay? There is no close icon and I am not getting any errors:
https://www.sm-55.de/reinigungsmittel-pflegemittel/fussbodenpflege/hart-bohnerwachs-rotbraun-fluessig-10l

**Comment by thdoan** [comment]
Hmm...the page you linked to is not using the mobile plugin. The mobile
plugin experience is like this:

https://thdoan.github.io/magnify/demo-mobile.html

**Comment by benlau87** [comment]
But in the demo I am having the same issue. Not always the first time, but happens nearly every time. I guess its dependant on the "swipe technique" or even mobile device?

**Comment by thdoan** [comment]
What exactly is the issue, you're not seeing the "x" icon in the overlay?

Are you even seeing the overlay? If you see the circular lens, then that's
not the mobile overlay -- that's the desktop version.

**Comment by benlau87** [comment]
I am not getting the mobile overlay then. Phone: Xiaomi Mi 5, Android 8 (MIUI Global 10.2 Stable)

**Comment by thdoan** [comment]
Thanks for reporting this. Can you see if you can reproduce the issue in
Chrome by using the device emulation feature? I don't have a Xiaomi phone
so let's try to reproduce it this way instead.

**Comment by pubdev1** [comment]
Hi,

Same problem here.

The problem is not the overlay. The problem is you can't touch your screen to scroll down if the picture is full screen (not the overlay, le picture). So, the magnify js takes place instead of the scroll.
